src/emulate_blue_2.2.5.c

```diff
@@ -1,10 +1,18 @@
 #include <err.h>
 #include <stdio.h>
 
+#include "bolos/cx.h"
 #include "bolos/cx_aes.h"
+#include "bolos/cx_hmac.h"
 #include "bolos_syscalls_blue_2.2.5.h"
 #include "emulate.h"
 
+unsigned long sys_os_perso_derive_node_bip32_seed_key(
```

Comment:
This prototype is already declared in `emulate_1.2.c`, but it isn't really appropriate. Please move these prototypes to `bolos/os_bip32.h`, that looks better to me.

Reply:
This also required to move the aliased declaration of `sys_os_perso_derive_node_bip32_seed_key()` to `bolos/os_bip32.c` and to add a non-aliased declaration of that function in `emulate.h` (where all sys_os_xxx() functions are declared) for the build to succeed. I verified that both `sys_os_perso_derive_node_bip32_seed_key()` and `sys_os_perso_derive_node_with_seed_key()` symbols exist only once in the built binary and that they refer to the same address:
```
(hsmenv) ~/work/dev/speculos ~ nm build/src/launcher | grep -E 'sys_os_perso_derive_node_.+_seed_key' 
000202a9 W sys_os_perso_derive_node_bip32_seed_key
000202a9 T sys_os_perso_derive_node_with_seed_key
```